docs/DeveloperGuide.md

```diff
@@ -335,6 +335,25 @@ Use case ends.
 
 Use case resumes at step 2.
 
+**Use case: F1 - Find student by name**
+
+**Actor: User (Tutor)**
```

Comment:
I think there might not be a need to state the actor in this use case, since we defined above that the actor is the user unless specified otherwise

Reply:
ok